### PR TITLE
Deprecate gist:_unitedNations

### LINKED ISSUE #207: Deprecate _unitedNations
Doesn't belong in an upper ontology, and if someone needs to reference the United Nations, an external URI can be used.

I'm calling this a patch because we could consider its inclusion an error, in a sense. It's a debatable point, however.

### PR DESCRIPTION
**Pull request by rjyounes**
Fixes #207

### CODE REVIEW

File: docs/ReleaseNotes.md [hunk 1]
```diff
@@ -44,6 +44,13 @@ Release X.x.x
 - Motivation/Rationale: Standardize release note format for readability.
 - Issue: [#338](https://github.com/semanticarts/gist/issues/338).
 
+*Deprecated `gist:_unitedNations`*
+
+- Deprecated `gist:_unitedNations` and moved from `gistCore` to `gistDeprecated`. Replaced the restriction on `gist:CountryGovernment` that it must be recognized by the United Nations with a restriction that it must be recognized by some Organization.
```
Comment (uscholdm) [inline docs/ReleaseNotes.md:49]: This explanation is correct, but a bit wordy. I wonder if a shorter style would be better. E.g.

Changed the restriction on CountryGovernment from
- `gist:recognizedBy value gist:_unitedNations` to
- `gist:recognizedBy some gist:Organization`

Comment (sa-bpelakh) [reply docs/ReleaseNotes.md:49]: That's a very OWL-y explanation, I think we should keep release notes to simple English (even if it is not quite as terse).
